src/stk500v2.c

```diff
@@ -1270,10 +1278,10 @@ static int stk500v2_initialize(const PROGRAMMER *pgm, const AVRPART *p) {
       unsigned char varef_read;
       stk500v2_getparm(pgm, PARAM_VADJUST, &varef_read);
       if (PDATA(pgm)->varef_get)
-        msg_info("Analog reference voltage value read as %.2fV\n", (varef_read / 10.0));
+        msg_info("Analog reference voltage value read as %.2f V\n", (varef_read / 10.0));
       // STK500: Write analog reference voltage
       else {
-        msg_info("Changing analog reference voltage from %.2f to %.2fV\n",
+        msg_info("Changing analog reference voltage from %.2f V to %.2f V\n",
```

Comment:
`unsigned char varef_read` above should be initialised to 0 (same reason as below, four times in this code)